automation/src/pages/mixin/NavigationMenu.ts

```diff
@@ -0,0 +1,156 @@
+import {ElementHandle, Page} from 'puppeteer-core';
+
+/**
+ * xpath selectors
+ */
+const selectors = {
+  root: '//',
+  hamburgerLink: '//div[contains(@style,\'transform: rotate(90deg)\') or contains(@style,\'transform: rotate(0deg)\')]/*[@shape=\'bars\']',
```

Comment:
These don't look stable to me. We should have further discussion about how we want to select elements

Reply:
To add to this: If it's not straightforward to write a text-based selector for an element, it means that element is probably inaccessible to someone using e.g. a screen reader. This is a good signal that we should probably add an `aria-label` or some other text labeling here.